src/main/notifications/index.test.ts

```diff
@@ -184,7 +184,7 @@ describe('main/notifications', () => {
                 value: 'darwin',
             });
 
-            getDarwinDoNotDisturb.mockReturnValue(true);
```

Comment:
Did this test pass before with the issue we had?

Reply:
Yes, because we were mocking the library. We unfortunately can't really test the behaviour of notifications since we don't have a library capable of verifying that the OS received them (that I know of).

Reply:
Ah ok, makes sense. There is so much that we are really testing with mocks 🤷‍♀️